src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj

```diff
@@ -75,6 +75,10 @@
     <None Include="**\windows\**\*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
```

Comment:
do we need to include ```<PackageReference Include="Microsoft.Graphics.Win2D" Version="1.0.0.30" />``` as well?

Reply:
TBH I don't if we need it on the lib. I'll try to find something

Reply:
No, we don't need it, at least not for now. As you can see in the readme is just an API to draw 2D graphics.
- https://github.com/microsoft/Win2D